modules/issue_tracker/ajax/EditIssue.php

```diff
@@ -0,0 +1,619 @@
+WHERE (u.CenterID=:CenterID) OR (u.CenterID=:DCC)",
+            array('CenterID' => $CenterID,
+                'DCC' => $DCCID)
+        );
+    }
+
+    foreach ($assignee_expanded as $a_row) {
+        $assignees[$a_row['UserID']] = $a_row['Real_name'];
+    }
+
+    $otherWatchers = array();
+    $potential_watchers_expanded = $db->pselect(
+        "SELECT Real_name, UserID FROM users",
+        array()
+    );
+    foreach ($potential_watchers_expanded as $w_row) {
+        $otherWatchers[$w_row['UserID']] = $w_row['Real_name'];
+    }
+
+    //can't set to closed if not developer.
+    if ($user->hasPermission('issue_tracker_developer')) {
+        $statuses = array(
+            'new' => 'New',
+            'acknowledged' => 'Acknowledged',
+            'assigned' => 'Assigned',
+            'feedback' => 'Feedback',
+            'resolved' => 'Resolved',
+            'closed' => 'Closed',
+        );
+    } else {
+        $statuses = array(
+            'new' => 'New',
+            'acknowledged' => 'Acknowledged',
+            'assigned' => 'Assigned',
+            'feedback' => 'Feedback',
+            'resolved' => 'Resolved',
+        );
+    }
+
+    $priorities = array(
+        'low' => 'Low',
+        'normal' => 'Normal',
+        'high' => 'High',
+        'urgent' => 'Urgent',
+        'immediate' => 'Immediate',
+    );
+
+    $categories = array(
+        'Behavioural Instruments' => 'Behavioural Instruments',
+        'Behavioural Battery' => 'Behavioural Battery',
+        'Data Entry' => 'Data Entry',
+        'Database Problems' => 'Database Problems',
+        'Examiners' => 'Examiners',
+        'SubprojectID/Project/Plan Changes' => 'SubprojectID' .
+            '/Project/Plan Changes',
+        'Imaging' => 'Imaging'
+    );
+
+    $modules = array();
+    $modules_expanded = $db->pselect(
+        "SELECT DISTINCT Label, ID FROM LorisMenu ORDER BY Label",
+        []
+    );
+    foreach ($modules_expanded as $m_row) {
+        $modules[$m_row['ID']] = $m_row['Label'];
+    }
+
+    //Now get issue values
+    $issueData = null;
+    if (!empty($_GET['issueID'])) { //if an existing issue
+        $issueID = $_GET['issueID'];
+        $issueData = $db->pselectRow(
+            "SELECT i.*, c.PSCID, s.Visit_label FROM issues as i " .
+            "LEFT JOIN candidate c ON (i.candID=c.CandID)" .
+            "LEFT JOIN session s ON (i.sessionID=s.ID) " .
+            "WHERE issueID = $issueID",
+            []
+        );
+        $issueData['commentHistory'] = getComments($issueID);
+        $issueData['whoIsWatching'] = getWatching($issueID);
+
+    } else { //just setting the default values
+        $issueData['reporter'] = $user->getData('UserID');
+        $issueData['dateCreated'] = date('Y-m-d H:i:s');
+        $issueData['centerID'] = $user->getData('CenterID');
+        $issueData['status'] = "new";
+        $issueData['priority'] = "low";
+        $issueData['issueID'] = null; //TODO: this is dumb
```

Comment:
This should be either set to 0, so adding issues works or not used at all. The default value for issueID is always the same and can be defined directly in JS.